### PR TITLE
Implemented get user route

### PR DESCRIPTION
**Pull request by aidansunbury**
Added the api route to get a user.

### CODE REVIEW

File: src/app/api/user/[id]/route.ts [hunk 1]
```diff
@@ -1 +1,18 @@
 // /api/user/[id] route
+import { NextResponse, NextRequest } from "next/server";
+import { PrismaClient } from "@prisma/client";
+const prisma = new PrismaClient();
+
```
Comment (Codify-Berkeley) [inline src/app/api/user/[id]/route.ts:5]: Make sure to add swagger documentation

File: src/app/api/user/[id]/route.ts [hunk 2]
```diff
@@ -1 +1,18 @@
...
+export async function GET(
+  request: NextRequest,
+  { params }: { params: { id: string } },
+) {
+  const { id } = request.id;
```
Comment (Codify-Berkeley) [inline src/app/api/user/[id]/route.ts:10]: Nice object destructuring!

File: src/app/api/user/[id]/route.ts [hunk 3]
```diff
@@ -1 +1,18 @@
...
+  const user = prisma.user.findUnique({
+    where: {
+      id: id,
+    },
+  });
+
+  return NextResponse.json(user, { status: 200 });
```
Comment (Codify-Berkeley) [inline src/app/api/user/[id]/route.ts:17]: Be sure to add additional response statuses, we don't want to return status 200 if the query fails

**Review by Codify-Berkeley — CHANGES_REQUESTED** [review]

**Review by Codify-Berkeley — APPROVED** [review]

**Comment by aidansunbury** [comment]
This route may not end up being used at all. I simply created it as an example to demonstrate how to do a code review. It does work however and can be used as an example when implementing other routes in the meantime.

**Pull request merged**